### PR TITLE
Uniformity 2 : Electric Boogaloo

### PR DESCRIPTION
**Pull request by alsoandanswer**
changes:
  - tweak: "Idris alt labcoat has big logo again."
dreammaker_Ok2oNMppVD

  - tweak: "Zeng-Hu labcoat highlights have been made more vibrant."
 dreammaker_jB4kGCR3nu

  - rscadd: "Adds surplus security uniforms. They're the old Aurora corporate uniforms, tweaked to match the new uniforms."
  Also, I added some Idris ones, I guess. The old ones are near-identical to the ones we already have.
  
dreammaker_iefImuE5ag

  - rscadd: "Adds, or re-adds, new hats and berets to match the old Aurora corporate uniforms."
  - tweak: "Collapses the faction hat selections to be neater. Please check your loadouts!"
  - tweak: "Undoes the swapping of officer and warden uniforms, and tweaks the other uniforms accordingly."
  - rscadd: "Zavodskoi scrubs made blacker, original black scrubs renamed to PMCG grey scrubs."

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by RyverStyx** [comment]
Why can't the Zeng-Hu lab coat be white instead of off-grey?

**Comment by alsoandanswer** [comment]
> Why can't the Zeng-Hu lab coat be white instead of off-grey?

....What? All labcoats are the same color of white.

**Comment by Caelphon** [comment]
It seems the clothes were reverted to Necropolis? Anyway, I’d greatly appreciate if it was instead rename to be Zavod or Zavodskoi. Thanks!

**Comment by alsoandanswer** [comment]
> It seems the clothes were reverted to Necropolis? Anyway, I’d greatly appreciate if it was instead rename to be Zavod or Zavodskoi. Thanks!

Addressed

**Comment by alsoandanswer** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 1]
```diff
@@ -110,18 +138,46 @@
...
 
-/datum/gear/faction/pmc_cap
-	display_name = "PMCG cap"
-	path = /obj/item/clothing/head/softcap/pmc
+/datum/gear/uniform/surpluspmc/New()
+	..()
+	var/list/surpluspmc = list()
+	surpluspmc["surplus EPMC security uniform"] = /obj/item/clothing/under/surplus/epmc
+	surpluspmc["surplus EPMC security uniform, alt"] = /obj/item/clothing/under/surplus/epmc/alt
+	surpluspmc["surplus PMCG security uniform"] = /obj/item/clothing/under/surplus/pmc
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_factions.dm:154]: These are EPMC uniforms, not PMCG. Rename the display name to be EPMC uniforms.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by NonQueueingMatt** [comment]
Not relevant anymore because of the new PRs.

**Pull request closed without merging**